packages/wrangler/src/api/integrations/platform/index.ts

```diff
@@ -103,27 +111,37 @@ export async function getPlatformProxy<
 >(
 	options: GetPlatformProxyOptions = {}
 ): Promise<PlatformProxy<Env, CfProperties>> {
+	const experimentalRemoteBindings = !!options.experimental?.remoteBindings;
+
 	const env = options.environment;
 
 	const rawConfig = readConfig({
 		config: options.configPath,
 		env,
 	});
 
-	const miniflareOptions = await run(
-		{
-			MULTIWORKER: false,
-			RESOURCES_PROVISION: false,
-			// TODO: when possible remote bindings should be made available for getPlatformProxy
-			REMOTE_BINDINGS: false,
-		},
-		() => getMiniflareOptionsFromConfig(rawConfig, env, options)
+	let remoteProxySession: RemoteProxySession | undefined = undefined;
+	if (experimentalRemoteBindings && rawConfig.configPath) {
+		const a = await maybeStartOrUpdateRemoteProxySession(
+			rawConfig.configPath,
+			null
+		);
+		remoteProxySession = a?.session;
+	}
+
+	const miniflareOptions = await getMiniflareOptionsFromConfig(
```

Comment:
We should modify `getMiniflareOptionsFromConfig()` to pass through `remoteBindingsEnabled` to `buildMiniflareBindingOptions()`

Reply:
mh.... sorry I'm not following, can you clarify what you mean?

Basically I removed all the `remoteBindingsEnabled` arguments since all of them actually seems very redundant to me, if a function's caller is passing a remoteProxySessionConnectionString, then the fact that remote bindings is enabled should be implicit (otherwise no connection string would be passed), doesn't that sound ok to you?